### PR TITLE
feat(sublime-adapter-debug): version `0.3.3` + pixels debug

### PR DESCRIPTION
**Pull request by SublimeTheo**
## Goal

Have a debug adapter to provide to sublime clients.
We sent some debug pixels using `sublime.analytics.fire` new signature.

## Nota Bene

Our prebid adapter is unusable with **SafeFrame**.

## How to test

If you never used this repo : 
```sh
# In your env
git clone git@github.com:SublimeSkinz/Prebid.js.git
```
If you're not up to date `npm install` in Prebid.js folder

To check coding style
```
gulp lint
```

To check our unit tests with Karma :
```
gulp test
```

To check coverage :
```
gulp test-coverage
```

- you could have an ad (refresh)

## Critical barometer

- :ok_hand: code safe, none or little impact on live applications

### CODE REVIEW

File: modules/sublimeBidAdapter.js [hunk 1]
```diff
@@ -125,7 +125,10 @@ export const spec = {
       data: {
         prebid: 1,
         request_id: requestId,
-        z: SUBLIME_ZONE
+        z: SUBLIME_ZONE,
+        meta: {
+          mediaTypes: bid.mediaTypes
```
Comment (sublimefengyu) [inline modules/sublimeBidAdapter.js]: Does mediaTypes always exist ?
Maybe bid.mediaTypes || {} is safer ?

File: modules/sublimeBidAdapter.js [hunk 2]
```diff
@@ -134,22 +137,40 @@ export const spec = {
...
 
     if (response) {
+      // Setting our returned sizes object to default values
+      let returnedSizes = {
+        width: 1800,
+        height: 1000
+      };
+
+      // Getting banner sizes, if not defined we set bannerSize to false
+      const bannerSize = bidRequest ? bidRequest.data.meta.mediaTypes.banner.sizes[0] || false : false;
```
Comment (sublimefengyu) [inline modules/sublimeBidAdapter.js]: We need tests here, the case when banner is null or when bannerSize = false.

File: modules/sublimeBidAdapter.js (hunk 1 above)
Comment (SublimeLeo) [reply modules/sublimeBidAdapter.js]: We may change `{ meta: { mediaType: { banner: sizes... } } }` to something like `{ w: width, h: height }`.
It's easier to read than a stringified json and is cheaper in data transfer.

File: modules/sublimeBidAdapter.js (hunk 2 above)
Comment (SublimeLeo) [reply modules/sublimeBidAdapter.js]: This should not be the case anymore since bidRequest now use data.w and data.h which are always set from buildRequest()
